### ISSUE TITLE
Add setting for scaling

### ISSUE THREAD

**Issue by Maingron** [Maingron/MainOS#10]
There should be a setting that allows the user to scale the interface. To be more precise, I think there should be 2 settings. 

- one for scaling text and images like on Windows XP, for example (I think it was called DPI scaling). This could be done through something like `body { font-size: initial * scaling }`
- one to scale the entire interface like you would do on Windows 10 for 4k screens - You'd probably set it to 200%. This should be done through `transform: scale(0.1 - 8)`

This will allow MainOS to work on very small screens. Also it's just more customisation for bigger screens.
Maybe change the defaults depending on the screen size when starting up or maybe even add a setting to scale some things automatically relative to the desired scaling.
